robottelo/constants/__init__.py

```diff
@@ -1765,6 +1765,8 @@
 CLOSED_STATUSES = ("ON_QA", "VERIFIED", "RELEASE_PENDING", "CLOSED")
 WONTFIX_RESOLUTIONS = ("WONTFIX", "CANTFIX", "DEFERRED")
 # Jira statuses used by Robottelo issue handler.
+JIRA_TESTS_PASSED_LABEL = "tests-passed"
+JIRA_TESTS_FAILED_LABEL = "tests-failed"
```

Comment:
@JacobCallahan @jyejare do you have any suggestions for the labels? For ex. `automation-passed`/`automation-failed` or `automation-tests-passed` or `automation-tests-failed`.

Reply:
I think as discussed in the past `automation` is a reserved keyword for ansible and in general `test` could have multiple meanings including unit tests, so maybe DEVs wants to push the results tomorrow for unit tests they will have to use a separate label.

Hence I would suggest `qe-passed` / `qe-failed` which covers al QE tests types and refers to QEs tests. Open for suggestions through.

Reply:
@jyejare I can understand the point for `automation` but the Dev one I'd argue about. Currently, we're the ones who need it and Dev's don't. And I'd not discard it based on future needs of Dev, which we're not even sure of. And yes, `test` could mean unit test but I don't think we need to differentiate labels based on that, at least not yet. 
As for `qe-passed` / `qe-failed`, this seems like it could have multiple meanings. I'd recommend something which has `test` in the name. For example, `qe-tests-passed` / `qe-tests-failed` or `robottelo-tests-passed` / `robottelo-tests-failed`

Reply:
@jameerpathan111 Its good to think ahead in some cases like this that would ask you to rethink / rework. Though we don't have yet from DEV, keeping that as a possibility would be a good path for us, needs no/less rethinking.

I respect your opinion to have `test` and hence I can vote for `qe-tests-*`.